compiler/noirc_evaluator/src/ssa/ir/types.rs

```diff
@@ -65,7 +65,8 @@ impl NumericType {
     ) -> Option<String> {
         match self {
             NumericType::Unsigned { bit_size } => {
-                let max = 2u128.pow(bit_size) - 1;
+                // The math here is a bit convoluted to avoid overflowing for u128
```

Comment:
We've generally used an if statement to just directly return `u128::MAX` in other cases.

Reply:
Ah, I considered doing that but I wasn't sure which way was better. I'll change it 👍